deployment/nginx.coffee

```diff
@@ -129,7 +129,9 @@ createLocations = (KONFIG) ->
       else
         auth = options.nginx.auth
 
-      if location.cors
+      if KONFIG.configName is 'dev'
+        cors = "add_header 'Access-Control-Allow-Origin' '*' always;"
```

Comment:
this might misdirect developers, some feature will work on their locals but it will be broken when they've pushed. just say'n.